src/command/Menus.js

```diff
@@ -1067,7 +1067,11 @@ define(function (require, exports, module) {
         }
 
         // Add a file-menu class to hide non-file items from file context menu
-        $("#project-context-menu").toggleClass("file-menu", mouseOrLocation.fileMenu);
+        if(mouseOrLocation.fileMenu) {
```

Comment:
Can you explain this change? It looks like it does exactly the same thing as the previous code. If it needs to be a boolean, there's no need of an `if/else`, just do:
```js
$("#project-context-menu").toggleClass("file-menu", !!(mouseOrLocation.fileMenu));
```